methods/safe.json

```diff
@@ -3,7 +3,7 @@
   "status": "registered",
   "verifiableDataRegistry": "Gnosis Safe",
   "contactName": "Gnosis Ltd.",
```

Comment:
This should be Safe Ecosystem Foundation. And officially the name was changed from `Gnosis Safe` to `Safe`